.github/workflows/ci.yml

```diff
@@ -51,9 +49,23 @@ jobs:
         if: ${{ matrix.scala == '3.0.0-M3' && matrix.platform == 'JVM' }}
         run: sbt ++${{ matrix.scala }}! testJVM
 
+  buildWebsite:
```

Comment:
this step is pointless as it doesn't get published anywhere. This just results in the build running slower. I would not add this.

Reply:
The point here is not to publish the website, but verify that it is buildable. Why shouldn't it be part of CI?

Reply:
@softinio Why shouln't building the website be part of CI? Or is it a process which is basically guaranteed to succeed? Or am I missing something?